src/FixedDataTable.js

```diff
@@ -957,8 +957,14 @@ var FixedDataTable = createReactClass({
       children.push(child);
     });
 
+    var tableWidth = props.width;
+    var reservedHeight = props.headerHeight + props.groupHeaderHeight + props.footerHeight;
```

Comment:
This should also take into account the border height and it's affected if horizontal scrollbar is visible.  There's a local variable called totalHeightReserved which already does this, but you may need to pull up to be above this logic.